test/core.jl

```diff
@@ -84,10 +87,13 @@ end
         @test all(predict(m3, X) .== y)
 
         mkdir("tmp")
-        write_model(m3, "tmp/test.h5")
-        m4 = load_model("tmp/test.h5")
-        rm("tmp", recursive=true)
-        compare_models!(m3, m4)
+        try
```

Comment:
You may want to use `tempdir()` instead.
http://docs.julialang.org/en/release-0.5/stdlib/file/#Base.tempdir